dealiiadapter.cc

```diff
@@ -378,14 +385,29 @@ class ElasticProblem
     Vector<double> old_displacement;
     Vector<double> displacement;
     Vector<double> old_forces;
+    Vector<double> forces;
     Vector<double> system_rhs;
+    //precice related initializations
+    int             meshID;
+    int             read_data_ID;
+    int             write_data_ID;
+    int             n_interface_nodes;
+
+    double          *read_data, *write_data;
```

Comment:
OK like this, but maybe it would be easier to understand if you always call them displacements and forces? For the moment, you are anyway restricted to this, right?

Reply:
I agree, use `data` only if you don't know what you are dealing with. Expressive names are always a plus.

Reply:
Also please use `std::vector` here. There is no reason to juggle with owning raw pointers.

Reply:
deal.ii has its own class for vectors. Should we still use the `std::vector` or stick to the deal.ii format?

Reply:
From their [docs](https://www.dealii.org/8.5.0/doxygen/deal.II/classVector.html#details).
> As opposed to the array of the C++ standard library called vector (with a lowercase "v"), this class implements an element of a vector space suitable for numerical computations.

So `std::vector` is the way to go here.